Modules/d2-drills/README.md

```diff
@@ -0,0 +1,27 @@
+# Week 2
```

Comment:
@brendanbohn you don't need to submit these drills as homework but if you want to; please move the files into your personal directory.